users/templates/user_notifications_list.html

```diff
@@ -1,36 +1,46 @@
 {% extends 'base.html' %}
```

Comment:
Please stick to our convention with 4 indents.

Reply:
done